### ISSUE TITLE
causal serial 在LifeEST复现

### ISSUE THREAD

**Issue by jinyuhang** [fajieyuan/sigir2020_peterrec#9]
请问，有人复现了 causal serial 在 LifeEST上的效果没？我尝试了各种方法都不得行。

PS：我用pytroch重写了PeterRec_noncau_parallel_classifier.py这个文件，只不过是negative sample 设置为99

**Comment by fajieyuan** [comment]
Hi，源代码PeterRec_noncau_parallel_classifier.py 有问题吗？我记得当时都应该测试过

**Comment by jinyuhang** [comment]
> Hi，源代码PeterRec_noncau_parallel_classifier.py 有问题吗？我记得当时都应该测试过

**Comment by jinyuhang** [comment]
> Hi，源代码PeterRec_noncau_parallel_classifier.py 有问题吗？我记得当时都应该测试过

我用pytorch参考PeterRec_noncau_parallel_classifier.py这个代码，写了causal_serial_classifier，在LifeEST数据上进行实验，复现不出论文Table2的实验结果。劳烦指正，我的复现如下：

（一）预训练NextitNet模型
  代码：参考NextitNet_TF_Pretrain_topk.py  
   数据集：lifestatus_pretrain_desent.csv
  参数：  lr 0.001,    batchsize 32,   epoch 6,  splite_rate 0.1,    embedding-dim 256,   negtive-smple 99,  使用sample softmaxt
  'cardinality': 1      'dilations': [1, 4, 1, 4, 1, 4, 1, 4, ],

(二)  微调
数据集：  lifestatus_finetune_desent.csv
数据划分: 训练集 0.7   验证集 0.03  测试集 0.27
medel_patch植入： 2 mp serial 
训练方式：PeterRec_noncau_parallel_classifier.py的训练方式，不过negtive_samples 设置成99
epoch：80

(3) 编程语言：pytorch
(4) 复现结果：HR@5   0.538（论文中为0.610）

我弄了好久，都没弄出来，劳烦前辈指正一下，若是方便能不能微信沟通(本人微信, 875526037)，非常感谢
（**在RecCold上复现出来了结果，结果略好与论文的。**）

**Comment by fajieyuan** [comment]
论文代码可以跑一下

**Comment by jinyuhang** [comment]
> 论文代码可以跑一下

就按照里面的默认的参数？

**Comment by fajieyuan** [comment]
论文里的参数哈

**Comment by jinyuhang** [comment]
> 论文里的参数哈

好的😊
